code/player/Inventory.cs

```diff
@@ -87,7 +87,7 @@ public bool Add(IItem item, bool makeActive = false)
         /// <returns></returns>
         public bool TryAdd(IItem item, bool makeActive = false)
         {
-            if (!Add(item, makeActive))
+            if (Owner.LifeState == LifeState.Dead || !Add(item, makeActive))
```

Comment:
@Alf21 Maybe we want to check for != LifeState.Alive here? 

There are `Dying, Respawnable, Respawning`, I wonder if we should avoid trying to give during these as well.

Reply:
yea, that should be the way to go